packages/loot-core/src/platform/server/sqlite/index.web.test.ts

```diff
@@ -27,7 +28,7 @@ describe('Web sqlite', () => {
 
     let rows = runQuery(db, 'SELECT * FROM numbers', null, true);
     expect(rows.length).toBe(1);
-    // @ts-expect-error Property 'number' does not exist on type 'unknown'
+    //@ts-expect-error: Property 'number' does not exist on type 'unknown'
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏would you mind undoing also these changes? Ideally the fingerprint of a PR should be as small as possible. And these comment changes are quite unnecessary. Thanks!
```suggestion
    // @ts-expect-error Property 'number' does not exist on type 'unknown'
```

Reply:
Oh, shoot, yep. Sorry.